lib/vanagon/platform/windows.rb

```diff
@@ -8,7 +8,10 @@ class Windows < Vanagon::Platform
       def generate_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         ["mkdir -p output/#{target_dir}",
-        "touch output/#{target_dir}/#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"]
+        "mkdir -p #{project.name}",
+        "cp #{project.name}.nuspec #{project.name}/",
+        "(cd #{project.name}; /cygdrive/c/ProgramData/chocolatey/choco pack)",
```

Comment:
To clarify @joshcooper's comment, Cygwin has a helper - `cygpath` that you can call with `cygpath C:/` and it will convert to `/cygdrive/c`. Josh was suggesting we have a method you pass all paths through (using `C:/` by default) that we could then have call `cygpath` to convert to a valid Cygwin path.

So something like
```
def normalize_path(p)
  `cygpath #{p}`
end

"(cd #{project.name}; #{platform.normalize_path('C:/ProgramData/chocolatey/choco`)} pack"
```